test/executor.test.js

```diff
@@ -468,9 +468,27 @@ describe('executor', function() {
 
     boot(app, appdir.PATH);
 
+    expect(Object.keys(require.cache)).to
+    .include(appdir.resolve('components/test-component/index.js'));
```

Comment:
Style: it's best to keep the chain of chai assertions on a single line. Also when a statement continues on a next line, the next lines should be indented by two spaces. 

```js
// either
expect(Object.keys(require.cache))
  .to.include(appdir.resolve('components/test-component/index.js'));

// or 
expect(Object.keys(require.cache)).to.include(
  appdir.resolve('components/test-component/index.js'));
```

The same comment applies to the test below.